src/shared/views/topMenu/TopMenuHeader/TopMenuHeader.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+import styles from './topMenuHeader.css'
+
+const TopMenuHeader = (props) => {
+  const titleTransform = (location) => {
+    const title = location.split('/')
+    if (title[0] === '') return 'Home'
+    return title[0].charAt(0).toUpperCase() + title[0].slice(1)
+  }
+  return (
+    <div className={styles.header}>
+      <h1>{titleTransform(props.title)}</h1>
```

Comment:
We are passing down the `title` prop, but actually what is being passed is the pathname.
I would suggest that transforming the pathname into a title should happen at the component above and this simply displays the title.

The above component has the transform, and i'd call it something very obvious like `getTitleFromPath`.

Var names should always describe _exactly what they are_, not what they will be used for, or transformed into.